tools/cldr-apps/src/main/webapp/js/new/cldrErrorSubtypes.js

```diff
@@ -0,0 +1,218 @@
+"use strict";
+
+/**
+ * cldrErrorSubtypes: encapsulate functions for the "Error Subtypes" page of Survey Tool
+ *
+ * Use an IIFE pattern to create a namespace for the public functions,
+ * and to hide everything else, minimizing global scope pollution.
+ */
+const cldrErrorSubtypes = (function () {
+  const instructions =
+    "<p>\n" +
+    "<i>Instructions</i>: This page shows the status of the subtype-to-URL mapping data.\n" +
+    "Each line here shows the CLDR error subtypes.<br />\n" +
+    "<b>Code</b> - this is the code <br/>\n" +
+    "<b>url</b> - this is the URL specified in the subtypeMapping.txt file<br/>\n" +
+    "<b>Status</b> - this shows whether the URL was fetched successfully. (200 indicates success.)\n" +
+    "Click the 'reload' 🔄 button to re-check the URL.\n" +
+    "</p>\n";
+
+  const coupleSeconds = 2000;
+  const redirectSoon = "<p>(Redirect in a couple seconds)</p>";
```

Comment:
shouldn't be a redirect in a JS context.

Reply:
I see, "Reload" would be better wording -- we can fix that later